core/nginx/conf/nginx.conf

```diff
@@ -275,6 +277,24 @@ http {
     include /etc/nginx/conf.d/*.conf;
 }
 
+stream {
+    # managesieve with implicit TLS.
+    # we can't do explicit since dovecot doesn't have the certificates
+    # It's arguable whether we should use the standardized port number for doing something non-standard
```

Comment:
I think it's fine. Clients should show an error.
Other proxies can detect TLS on the fly, so we could add starttls when a proxy is implemented.

Reply:
OK cool.